api/src/errors/routes.py

```diff
@@ -0,0 +1,12 @@
+from flask import make_response
+
+from src.errors import blueprint
+from src import metrics
+
+@blueprint.app_errorhandler(404)
```

Comment:
What's the reason for this change?

Reply:
that we use a blueprint instead of decorating the app instance? there are several reasons:
- we prevent circular imports at the current app architecture
- we do not have get duplicated timeseries with prometheus
- it fits with the modular architecture of our app